src/component/mxgraph/shape/activity-shapes.ts

```diff
@@ -135,4 +132,16 @@ export class SubProcessShape extends BaseActivityShape {
     xmlCanvas.setDashed(StyleUtils.isDashed(this.style), StyleUtils.getFixDash(this.style));
     xmlCanvas.setDashPattern(StyleUtils.getDashPattern(this.style));
   }
+
+  public paintForeground(c: mxAbstractCanvas2D, x: number, y: number, w: number, h: number): void {
+    super.paintForeground(c, x, y, w, h);
+
+    if (StyleUtils.getBpmnIsExpanded(this.style) === 'false') {
+      const subProcessKind = StyleUtils.getBpmnSubProcessKind(this.style);
+      if (subProcessKind === ShapeBpmnSubProcessKind.EMBEDDED) {
+        const paintParameter = IconPainter.buildPaintParameter((c as unknown) as mxgraph.mxXmlCanvas2D, x, y, w, h, (this as unknown) as mxgraph.mxShape, 0.17, false, 1.5);
```

Comment:
:question: same question about the `0.17` and `1.5` magic numbers

Reply:
Yes, I just test with different values.

Reply:
I calculate myself these numbers, so there is no comment to add.